.github/workflows/ci.yaml

```
@@ -256,16 +259,29 @@ jobs:
           echo "JENKINS_USER=${{ steps.parse_secrets.outputs.TOPTAL_TRIGGERBOT_USERNAME }}" >> $GITHUB_ENV
           echo "JENKINS_BUILD_TOKEN=${{ steps.parse_secrets.outputs.TOPTAL_TRIGGERBOT_BUILD_TOKEN }}" >> $GITHUB_ENV
 
+      - name: Get toptal/actions
+        uses: actions/checkout@v4
+        with:
+          repository: toptal/actions
+          token: ${{ steps.parse_secrets.outputs.TOPTAL_REPOACCESSBOT_TOKEN }}
+          path: ./.github/actions/
+      
+      - name: Replace toptal/actions/get-job-url@main from trigger-jenkins-job/action.yml
+        run: |
+          sed -i 's|toptal/actions/get-job-url@main|./.github/actions/get-job-url|' ./.github/actions/trigger-jenkins-job/action.yml
+        shell: bash
+
       - name: Trigger doc deployment job
-        uses: toptal/jenkins-job-trigger-action@1.0.1
+        uses: ./.github/actions/trigger-jenkins-job/
```

Do you think it makes sense to name it `jenkins_trigger` to be consistent with the approach in .github/workflows/release.yml?

```suggestion
        uses: ./.github/jenkins_trigger/trigger-jenkins-job/
```

---

@sashuk Basically we are checking-out the `toptal/actions` repo. So I checked it out in `.github/actions/ `[path](https://github.com/toptal/picasso/pull/4014/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR267)

---

@serbantarmure But why in the same situation in https://github.com/toptal/picasso/pull/4014/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R157 it is called `jenkins_trigger` then?

---

I forgot to change it :sweat_smile:

---

@serbantarmure Oh great, thanks! 🙂